### ISSUE TITLE
failed to run "make" on termux

### ISSUE THREAD

**Issue by dijkstrabc** [LostRuins/koboldcpp#700]
![20240220163755](https://github.com/LostRuins/koboldcpp/assets/110527543/6b2d3f94-3b36-492a-aac4-3fc329ff57ea)

it is an android snapdragon 8gen3 with 16gb ram, the building process turns out error with these post on the pic, python 3.11.8 is installed

**Comment by LostRuins** [comment]
Yes there was a bug in the code. Its been fixed in the experimental branch. 

Try 

`git checkout concedo_experimental`

Then `git pull` and  `make`

**Comment by dijkstrabc** [comment]
> Yes there was a bug in the code. Its been fixed in the experimental branch.
> 
> Try
> 
> `git checkout concedo_experimental`
> 
> Then `git pull` and `make`

problem solved, thanks

**Comment by LostRuins** [comment]
Should be fixed in stable now
